kubemarine/procedures/check_paas.py

```diff
@@ -870,6 +873,17 @@ def container_runtime_configuration_check(cluster: KubernetesCluster) -> None:
                 cluster.log.debug(yaml.safe_dump(yaml.safe_load(diff.to_json())))
                 success = False
 
+            actual_registries_conf = actual_registries.get(node.get_host(), {})
+            for registry, expected_registry in expected_registries.items():
+                diff = DeepDiff(expected_registry, actual_registries_conf.get(registry))
```

Comment:
Do we need to check when some nodes have registries that are not present / enriched in cluster.yaml?

Reply:
I thought and decided, that yes. 
Benefits:
1. Backward compatibility with current behavior;
2. Reproducible: configuration is depended only from kubemarine specification;

Fixed `check_paas` to check both sites;
Added cleaning `config_path` directory;